commit 0b03a90cb05c32a9ddb424dd35cbccd2729ec658 (refs/changes/97/1397/1)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-02-02T09:24:09+08:00 (2 years, 8 months ago)
    
    DAPI-336: Disk calc validation on quota assumes integers but in reality there can be decimals

diff --git a/package.json b/package.json
index 3f8f97d..d7a94c3 100644
--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#d7c29837103887a75ed1fb5fe84d1d6b89d0dae2",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#2eee9106262f15b6c9ef04cf68e80a29abe44273",
     "deep-equal": "1.0.1",
     "dox": "0.4.1",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
